src/chapters/classes-objects-math/class-type.rst

```diff
@@ -0,0 +1,116 @@
+
+Classes: A Primer
+=====================
+
+Now that you have a better understanding of methods, we should touch on classes and objects *briefly*.  
+These are big concepts in C#, and OOP in general.  
+This chapter is designed to help you get familiar with the larger concepts of classes and objects.
+We will explore definitions and walkthrough examples that will demonstarate how classes, objects, and methods work together.
+
+
+Class Type
+-------------------
+
+Quick refesher on data types.  As discussed in chapter 4, C# contains value types and reference types.  
```

Comment:
can you replace the reference to chapter 4 with a reference to the section title - and you can even use a :ref: link to give a hyperlink back to that page.
we want to avoid referring to chapter numbers because if we move sections around, we dont want to have to check the book for references like this